Toolset/libraries/revidelibrary.8.livecodescript

```diff
@@ -3534,26 +3534,60 @@ on revIDEActionGroupObjects
    end if
 end revIDEActionGroupObjects
 
-on revIDEGroupObjects pObjectIDs
-   local tLayer, tGroupID, tList, tObj
-   
-   put the long id of (line 1 of pObjectIDs) into tObj
-   put 1000000 into tLayer
 
-   lock screen
+/**
 
-   repeat for each line tControl in pObjectIDs
-      if the layer of tControl < tLayer then put the layer of tControl into tLayer
-   end repeat
+Name: revIDEGroupObjects
+
+Type: command
+
+Syntax: revIDEGroupObjects <objectList>
+
+Summary: Group a list of objects in their layer order
 
-   // PM-2016-04-07: [[ Bug 17301 ]] Use the "group" command (without an objectList)
-   // so as to group the selection in layer order
-   group
+Description:
+Group a list of objects in the order they are layered in
+
+Parameters:
+objectList (string): A return delimited list of long object IDs
+
+The result:
+The new group long ID or empty in the case the group could not becreated
+
+**/
+on revIDEGroupObjects pObjectIDs
+   filter lines of pObjectIDs without empty
+   
+   if pObjectIDs is empty then
+      return empty
+   end if
+   
+   local tChar
+   repeat with tChar = the number of chars of pObjectIDs down to 1
+      if char tChar of pObjectIDs is return then
+         delete char tChar of pObjectIDs
+      else
+         exit repeat
+      end if
+   end repeat
```

Comment:
Yes of course it does! GitHub really needs a facepalm reaction ;-) hacking away at autocomplete on the flights so I'll sort this PR out when I'm home.